O/OpenSpiel/build_tarballs.jl

```diff
@@ -10,58 +10,89 @@ delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10"]
 
 name = "OpenSpiel"
-version = v"1.1.1"
+version = v"1.2"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/deepmind/open_spiel/archive/v1.1.1.tar.gz", "73d46e2d9ce7a86a420aad141b789e98237463b5c98608401faf86b15ec5eaf9"),
-    ArchiveSource("https://github.com/findmyway/dds/archive/v0.1.1.tar.gz", "fd13ee77feb5b5c3dfcc3333a0523266beb2a3d27715703cf508313af25306e5"),
-    ArchiveSource("https://github.com/abseil/abseil-cpp/archive/20211102.0.tar.gz", "dcf71b9cba8dc0ca9940c4b316a0c796be8fab42b070bb6b7cab62b48f0e66c4"),
-    ArchiveSource("https://github.com/findmyway/hanabi-learning-environment/archive/v0.1.0.tar.gz", "6126936fd13a95f8cadeacaa69dfb38a960eaf3bd588aacc8893a6e07e4791a3"),
-    ArchiveSource("https://github.com/findmyway/project_acpc_server/archive/v0.1.0.tar.gz", "e29f969dd62ba354b7019cae3f7f1dbfbd9a744687ea4a8f7494c2bb1ee87382"),
+    GitSource("https://github.com/deepmind/open_spiel.git", "9e4559bd1d794aa0b6476013464b58329d8f2b51"), # v1.2.0
+    GitSource("https://github.com/findmyway/dds.git", "091ea94358a4016d4fb6069dea5c452cdc98d0bd"), # v0.1.1
+    GitSource("https://github.com/abseil/abseil-cpp.git", "b971ac5250ea8de900eae9f95e06548d14cd95fe"), # 20230125.2
+    GitSource("https://github.com/findmyway/hanabi-learning-environment.git", "b31c973e3930804b9e27d1a20874e08d8643e533"), # v0.1.0
+    GitSource("https://github.com/findmyway/project_acpc_server.git", "de5fb88ac597278c96875d3c163ce71cdfe7ea79"), # v0.1.0
+    DirectorySource("./bundled"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.14.sdk.tar.xz", "0f03869f72df8705b832910517b47dd5b79eb4e160512602f593ed243b28715f"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+# This requires macOS 10.14
+if [[ "${target}" == x86_64-apple-darwin* ]]; then
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    popd
+fi
 
-mv open_spiel-* open_spiel
-mv abseil-cpp-* open_spiel/open_spiel/abseil-cpp
-mv dds-* open_spiel/open_spiel/games/bridge/double_dummy_solver
-mv hanabi-learning-environment-* open_spiel/open_spiel/games/hanabi/hanabi-learning-environment
-mv project_acpc_server-* open_spiel/open_spiel/games/universal_poker/acpc
+# Apply abseil patch to fix -march issue
+cd abseil-cpp
+atomic_patch -p1 ../abseil.patch
+cd ../
+
+macosflags=
+case $target in
+  *apple-darwin*)
+    macosflags="-DCMAKE_CXX_COMPILER_ID=AppleClang"
+    macosflags="$macosflags -DCMAKE_CXX_COMPILER_VERSION=10.0.0"
+    macosflags="$macosflags -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=11"
```

Comment:
This looks wrong: the compiler is _not_ Apple Clang, and the variable you should set for the standard is called [`CMAKE_CXX_STANDARD`](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD).  Also, why are these flags needed at all?  CMake should figure out on its own the compiler name and version, and the standard version should be set by the upstream project, if you have to do that because something it fails then it's an upstream bug.

Reply:
I copied this (blindly) from https://github.com/JuliaPackaging/Yggdrasil/blob/c91eb808489a50fa4cc2e3d2dcdadcc3b51c8f9c/L/libcgal_julia/build_tarballs.jl#L36, let me see if it builds without it.